core/utils.js

```diff
@@ -622,3 +622,37 @@ Blockly.utils.screenToWsCoordinates = function(ws, screenCoordinates) {
   var finalOffsetMainWs = finalOffsetPixels.scale(1 / ws.scale);
   return finalOffsetMainWs;
 };
+
+/**
+ * Builds an object structure for the provided namespace path, ensuring that
+ * names that already exist are not overwritten. For example:
+ * "a.b.c" -> a = {};a.b={};a.b.c={}
+ * @param {string} name name of the object that this file defines.
+ * @param {*=} opt_object the object to expose at the end of the path.
+ */
+Blockly.utils.exportSymbol = function(name, opt_object) {
```

Comment:
Find these comments in our codebase:
// Export symbols that would otherwise be renamed by Closure compiler.

And either replace them with exportSymbol, or delete this function and use the existing approach.  Whatever we do, it should be consistent.

Reply:
Nice, done.